config.js

```diff
@@ -1,4 +1,4 @@
-var config = {
+const config = {
```

Comment:
Would you mind leaving this as a `var` please? I muck around with it for testing.